src/tests/Microsoft.PowerFx.Core.Tests.Shared/UserDefinedTypeTests.cs

```diff
@@ -126,6 +135,23 @@ public void TestUDTErrors(string typeDefinition, int expectedErrorCount, string
             Assert.All(errors, e => Assert.Contains(expectedMessageKey, e.MessageKey));
         }
 
+        [Theory]
+
+        // not in Type literal expression;
+        [InlineData("Points := Type([{ x: Number, y: Number }]); F(): Points = RecordOf(Points); ", "ErrknownTypeHelperFunction")]
+        [InlineData("Points := Type([{ x: Number, y: Number }]); F(): Number = RecordOf(Points).x; ", "ErrknownTypeHelperFunction")]
```

Comment:
> Type([{ x: Number, y: [](http://example.com/codeflow?start=30&length=22)

Is following valid? Can you add a testcase for it?

T1 := Type(RecordOf(Type([{A:Number}]))

Reply:
No it is not . we only allow FirstNames within `RecordOf` as inlining types within is just redundant